server/routes/__tests__/money.spec.js

```diff
@@ -134,37 +146,32 @@ describe('Prisoner Money', () => {
     ],
   };
 
-  const fourOhFour = createAxiosRequestError('🤷‍♂️', null, 404);
-  const fiveOhThree = createAxiosRequestError('💥', null, 503);
-
-  const setMockUser = (req, res, next) => {
-    req.user = testUser;
-    next();
-  };
+  const currentUser = jest.fn();
 
   beforeEach(() => {
     jest.clearAllMocks();
+    currentUser.mockReturnValue(testUser);
+
     app = setupBasicApp();
+    app.use((req, res, next) => {
+      req.user = currentUser();
+      next();
+    });
+    app.use('/money', moneyRouter);
+    app.use(consoleLogError);
```

Comment:
using `spy` and user injection means we can just do this once in a top level `beforeEach`